Kitodo-DataEditor/src/test/resources/log4j2.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Maybe this file belong to `src/main/resources` or a `log4j2.xml` file must be in this place too.